dashboard/src/components/resource-list/ResourceListView.test.js

```diff
@@ -14,14 +17,35 @@ import {
 
 configure({ adapter: new Adapter() });
 
-describe("ResourceListView", () => {
-  it("passes through title", () => {
-    const list = shallow(<ResourceListView title="test" title={"test"} />);
-    expect(list.children().props().title).toBe("test");
+export const ThemeWrapper = ({ children }) => (
```

Comment:
Why bother using the theme at all in these tests?

Reply:
The theme is needed because the component won't mount without the theme import, even though the theme has nothing to do with what is being tested.

Reply:
Why?